core/app/middleware_test.go

```diff
@@ -126,11 +185,24 @@ func assertRequestStatus(
 	}
 }
 
-func createEmptyRequest(t *testing.T, method, path string) *http.Request {
-	req, err := http.NewRequest(method, path, nil)
-	if err != nil {
-		t.Logf("Got an error while creating request: %v", err)
-		t.Fatal("There should be no error while createing the request")
```

Comment:
Was this replaced by another method?

Reply:
Ya actually, I made a little testing library in the `simtest` package (`github.com/flapflapio/simulator/internal/simtest`), so this function got replaced into a general-purpose utility function in that package.